t/object_methods.t

```diff
@@ -85,3 +85,11 @@ is($datetime3->minus_time_interval('1d')->is_same_as($datetime2),  1,          '
 is($datetime1->minus_time_interval(0),                             $datetime1, 'minus_time_interval(0) yields the same object');
 is($datetime2->minus_time_interval('-1d')->is_same_as($datetime3), 1,          'minus_time_interval("-1d") yields one day ahead.');
 throws_ok { $datetime3->minus_time_interval("one") } qr/Bad format/, 'minus_time_interval("one") is not a mind-reader..';
+
+subtest 'move_to_nth_dow' => sub {
+    is($datetime1->move_to_nth_dow(3, 'Wed')->day_of_month, 21, 'Third Wednesday of Dec 2012 is the 21st');
```

Comment:
Looking at a calendar, this is not true! The 21st is a Friday.  I will look into what is happening here.